driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionKmsTlsTest.java

```diff
@@ -120,6 +123,10 @@ public void testInvalidKmsCertificate() {
         }
     }
 
+    /**
+     * See <a href="https://github.com/mongodb/specifications/tree/master/source/client-side-encryption/tests#11-kms-tls-options-tests">
+     * 11. KMS TLS Options Tests</a>.
+     */
     @Test()
     public void testThatCustomSslContextIsUsed() {
```

Comment:
A new use-case related to named providers was recently added to the KMS TLS Options specification prose tests. While the existing tests for KMS TLS Options do not precisely align with the spec tests, they have been purposefully designed for simplicity and effectiveness. In line with the approach discussed in [this original comment](https://github.com/mongodb/mongo-java-driver/pull/820/files#r745607434), this test has been extended to include named providers.